4.9/alpha/pipeline-alpha.md

```diff
@@ -75,7 +75,9 @@ oc import-image redhat-marketplace-index \
 
 ### <a id="step5"></a>Step 5 - Install the Certification Pipeline and dependencies into the cluster
 ```bash
-oc apply -R -f ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml
+git clone git@github.com:redhat-openshift-ecosystem/operator-pipelines.git
+cd operator-pipelines
+oc apply -R -f ansible/roles/operator-pipeline/templates/openshift/pipelines
```

Comment:
this one replaced to have the entire dir, because there's a later step which runs pipeline-serial.yaml